vars/OpenBSD.yml

```diff
@@ -0,0 +1,13 @@
+---
+__fluentd_user: _fluentd
+__fluentd_group: _fluentd
+__fluentd_package_name: rubygem-fluentd
+__fluentd_service_name: fluentd
+__fluentd_config_dir: "/etc/fluentd"
+__fluentd_config_file: "{{ __fluentd_config_dir }}/fluent.conf"
```

Comment:
s/__fluentd_config_dir/fluentd_config_dir/ or users must modify fluentd_config_file when fluentd_config_dir is set.